### PR TITLE
Test coverage

### LINKED ISSUE #7: Improve test coverage
Get our test coverage from 60% or whatever it is now to 80%+

one of the main things missing from tests at this point is the integration with django-two-factor-auth / django-otp
Will need to load / instantiate the login view from `src/django_security_keys/ext/two_factor` somewhere and test the login process through that.

### PR DESCRIPTION
**Pull request by vegu**
fixes #7

### CODE REVIEW

File: tests/project/templates/two_factor/_base.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% include "site/header.html" %}
+{% include "site/view_header.html" with errors=errors %}
```
Comment (vegu) [inline tests/project/templates/two_factor/_base.html:2]: @jason20c can drop these

File: tests/project/templates/two_factor/_base.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+      {% endblock %}
+      </div>
+      {% endif %}
+    </div>
+  </div>
+</div>
+
+
+{% include "site/footer.html" %}
```
Comment (vegu) [inline tests/project/templates/two_factor/_base.html:57]: @jason20c can drop this

File: tests/project/templates/two_factor/core/backup_tokens.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{% extends "two_factor/_base_focus.html" %}
```
Comment (vegu) [inline tests/project/templates/two_factor/core/backup_tokens.html:1]: @jason20c entire file can be removed

File: src/django_security_keys/templatetags/util.py [hunk 4]
```diff
@@ -0,0 +1,273 @@
+import datetime
```
Comment (vegu) [inline src/django_security_keys/templatetags/util.py:1]: @jason20c entire file can be removed

File: tests/project/templates/two_factor/profile/profile.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{% extends "two_factor/profile/profile.html" %}
```
Comment (vegu) [inline tests/project/templates/two_factor/profile/profile.html:1]: @jason20c entire file can be removed

**Review by grizz — APPROVED** [review]

**Pull request merged**